src/components/Error/empty.styl

```diff
@@ -1,70 +1,32 @@
-@require 'cozy-ui'
+@import 'cozy-ui/'
```

Comment:
Why `import` and not `require`?

Reply:
Not quite sure exactly but apparently, due to our Stylus management, `@require` does not do what it's supposed to do. @ptbrowne knows more as it is one of its recommandations.

Reply:
i'd be interested in knowing more about this. IIRC `import` creates a copy, which is what was causing massive build times. And we use `require` everywhere else.

Reply:
Mmh, I recommended to use `require` instead of `import`, did you experience problems with `require` @GoOz  >